praw/models/reddit/emoji.py

```diff
@@ -0,0 +1,71 @@
+"""Provide the EMOJI class."""
+from ...const import API_PATH
+from .base import RedditBase
+
+
+class Emoji(RedditBase):
+    """An individual Emoji object."""
+
+    __hash__ = RedditBase.__hash__
+    STR_FIELD = 'name'
+
+    def __init__(self, reddit, subreddit, name, _data=None):
+        """Construct an instance of the Emoji object."""
+        self.reddit = reddit
+        self.subreddit = subreddit
+        self.name = name
+        super(Emoji, self).__init__(reddit, _data)
+
+    def add(self, filepath):
+        """Add an emoji to this subreddit.
+
+        :param emoji: An emoji name (e.g., ``'cake'``) or
+            :class:`~.Emoji` instance.
+
+        To add ``'cake'`` to the subreddit ``'praw_test'`` try:
+
+        .. code:: python
+
+           reddit.subreddit('praw_test').emoji['cake'].add('cake.png')
+
+        """
+        data = None
+        if filepath.lower().endswith('.png'):
+            data = {'filepath': filepath, 'mimetype': 'image/png'}
+        elif filepath.lower().endswith('.jpg') \
```

Comment:
Can we simplify this to assume (for now) that the file either either a png or jpg? E.g.,

    data = {'filepath': filepath, 'mimetype': 'image/jpeg'}
    if filepath.lower().endswith('.png'):
        data['mimetype'] = 'image/png'